notation.go

```diff
@@ -181,6 +188,9 @@ type RemoteVerifyOptions struct {
 	// will be processed for verification. If set to less than or equals
 	// to zero, an error will be returned.
 	MaxSignatureAttempts int
+
+	// UserMetadata contains key-value pairs that must be present in the signature
+	UserMetadata map[string]string
```

Comment:
We need to keep this option if we reuse `Annotations` of [ocispec.Descirptor](https://pkg.go.dev/github.com/opencontainers/image-spec/specs-go/v1#Descriptor).